back/src/domains/establishment/adapters/PgEstablishmentAggregateRepository.integration.test.ts

```diff
@@ -71,49 +68,6 @@ describe("PgEstablishmentAggregateRepository", () => {
 
   describe("Offers", () => {
     describe("searchImmersionResults", () => {
-      it("returns empty list when repo is empty", async () => {
-        expectToEqual(
-          await pgEstablishmentAggregateRepository.searchImmersionResults({
-            searchMade: cartographeSearchMade,
-          }),
-          [],
-        );
-      });
```

Comment:
ce test là je ne sais pas si c'est exprèt que tu le vire, mais les tests repo vide sont aussi important, ca permet d'éviter souvent des erreurs du style `select * from table where id in (...rien ici...)`

Reply:
Test juste déplacé ligne 402